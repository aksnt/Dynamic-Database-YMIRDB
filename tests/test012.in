SET a 1
SET b 4 5 a
SET c b a 1
SNAPSHOT
SET d 6 8
SNAPSHOT
CHECKOUT 1
LIST ENTRIES
BYE
