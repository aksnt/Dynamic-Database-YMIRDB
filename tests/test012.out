> ok

> ok

> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> c [b a 1]
b [4 5 a]
a 1

> bye
